readme.md

```diff
@@ -46,13 +46,29 @@ module.exports = {
   mongodbMemoryServerOptions: {
     binary: {
       version: '4.0.3',
-      skipMD5: true
+      skipMD5: true,
     },
     instance: {
-      dbName: 'jest'
+      dbName: 'jest',
+    },
+    autoStart: false,
+  },
+};
+```
+
+To use separate database for each jest worker pass the `separateMongoInstances: true` (doesn't create `process.env` variable when using this option):
```

Comment:
```suggestion
```suggestion
To use a separate database for each jest worker pass the `useSharedDBForAllJestWorkers: false` (doesn't create `process.env` variable when using this option):
```

```